src/main/resources/target/checkstyle-result.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do you need it?

Reply:
> Why do you need it?

folder removed. This was some after mvn clean package.